libs/wallet/src/connectors/rest-connector.ts

```diff
@@ -85,7 +85,7 @@ export class RestConnector implements VegaConnector {
     }
   }
 
-  async sendTx(body: TransactionSubmission) {
+  async sendTx(body?: TransactionSubmission) {
```

Comment:
question (blocking): why are we permitting empty transactions? These would immediately get rejected so I think this doesn't make and sense?